style.css

```diff
@@ -511,3 +519,225 @@ textarea:focus,
 input:focus {
   outline: none;
 }
+
+@media screen and (min-width: 768px) {
+  .mobile-menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu-items {
+    margin-top: 15px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 114px;
+  }
+
+  .menu-item a {
+    font-family: "Poppins", sans-serif;
+    font-style: medium;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    margin-right: 16px;
+    margin-left: 16px;
+    padding: 10px;
+  }
+
+  .home {
+    background-image: url(./image/Desktop-Images/desktopbg.svg);
+    background-size: contain;
+    background-position: center;
+    align-items: center;
+    margin-top: -46px;
+    margin-bottom: 76px;
+    margin-left: 34.6px;
+    margin-right: 8px;
+    height: 916px;
+  }
+
+  .mylogo {
+    margin-left: 145px;
+    margin-top: 25px;
+  }
+
+  .bio {
+    justify-content: center;
+  }
+
+  .title {
+    font-style: bold;
+    font-size: 48px;
+    line-height: 60px;
+    margin-left: 260px;
+    margin-right: 180px;
+    margin-top: 359px;
+    margin-bottom: 6px;
+    width: 920px;
+  }
+
+  .text {
+    font-size: 20px;
+    line-height: 28px;
+    margin-left: 260px;
+    margin-top: 6px;
+    padding-top: 15px;
+  }
+
+  .connect {
+    margin-left: 260px;
+    margin-top: 12px;
+  }
+
+  .social-icon {
+    margin-left: 270px;
+  }
+
+  .card {
+    flex-direction: row;
+    justify-content: center;
+    width: 1156px;
+    height: 496px;
+    border-radius: 24px;
+    gap: 30px;
+  }
+
+  .work {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 5px;
+  }
```

Comment:
- [ ] Kindly make sure the background color of the work section matches the Figma design as we have to stick to it :recycle: 
- Your design:
![Screenshot from 2023-03-02 17-51-48](https://user-images.githubusercontent.com/97834160/222479886-02d89b91-9c38-4afc-8cbb-8f70069138ce.png)

- Figma
![Screenshot from 2023-03-02 17-49-01](https://user-images.githubusercontent.com/97834160/222478672-0332cfe4-f039-4ca3-beef-7a85b452f1da.png)

Reply:
Hi @GedeonTS  , 

Thanks for your time and for reviewing my code. I have updated the background color of the work section according to the Figma template.